package.json

```diff
@@ -61,6 +60,9 @@
   "devDependencies": {
     "@bitwarden/cli": "^1.11.0",
     "@iconify/icons-ant-design": "^1.0.8",
+    "@iconify/icons-bytesize": "^1.0.3",
+    "@iconify/icons-mdi": "^1.0.149",
+    "@iconify/react": "^1.1.3",
```

Comment:
These should be in `dependencies`, fix this by running `npm i -S @iconify/icons-bytesize @iconify/icons-mdi @iconify/react` in terminal and commit again

Reply:
Resolved and added to dependencies. Thank you!!